charts/kubewarden-crds/Chart.yaml

```diff
@@ -15,7 +16,29 @@ maintainers:
   email: jguilhermevanz@suse.com
 - name: Raul Cabello Martin
   email: raul.cabello@suse.com
+keywords:
+  - Security
+  - Infrastructure
+  - Monitoring
+
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.0
+version: 1.1.10
```

Comment:
I would bump to `1.2.0`

Reply:
Done in amended commit https://github.com/kubewarden/helm-charts/pull/129/commits/c684e4f2e778d0725cad013bc4fcb429417f74e8 (there were more things to change)